app/models/team.rb

```diff
@@ -45,18 +45,23 @@ def visible
     end
 
     # Returns a base scope reflecting the relevant teams depending on what
-    # phase of the running season we're currently in.
+    # phase of the running season we're currentin_ly in.
     def by_season_phase
       if Time.now.utc > Season.current.acceptance_notification_at
-        Team.where season: Season.current, kind: %w(sponsored voluntary)
+        Team.in_current_season.selected
       else
-        Team.visible.where season: Season.current
+        Team.in_current_season.visible
       end
     end
 
-    def active_teams
-      Team.where(season: Season.current, kind: %w(sponsored voluntary)).order(:name)
+    def in_current_season
```

Comment:
I will, sure, but I did it on purpose. I like a little more verbosity to makes things more clear and distinctive. And hopefully more beginner friendly. From my experience from last year, I found the fact that different things have similar names the most confusing. But it is probably a Rails convention, isn't it?